src/collection.ts

```diff
@@ -1056,7 +1056,9 @@ export class Collection<TSchema extends Document = Document> {
    *
    * @remarks Only available when used against a 7.0+ Atlas cluster.
    */
-  listSearchIndexes(options?: ListSearchIndexesOptions): ListSearchIndexesCursor;
+  listSearchIndexes(
+    options?: Omit<ListSearchIndexesOptions, 'readConcern' | 'writeConcern'>
```

Comment:
Is there ever a scenario where `ListSearchIndexesOptions` will allow a read / write concern?  If not, maybe we should just omit these options on the `ListSearchIndexesOptions` interface instead

Reply:
IIUC, we can run ListSearchIndexes with runCommand outside of Atlas.